src/extension/webviews/help_view.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining
+ * a copy of this software and associated documentation files
+ * (the "Software"), to deal in the Software without restriction,
+ * including without limitation the rights to use, copy, modify, merge,
+ * publish, distribute, sublicense, and/or sell copies of the Software,
+ * and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be
+ * included in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+ * IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
+ * CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+ * TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+import * as vscode from 'vscode';
+import * as path from 'path';
+import {getWebviewHtml} from '../webviews';
+import {HelpMessageType, HelpPanelMessage} from '../message_types';
+import {WebviewMessageManager} from '../webview_message_manager';
+
+export class HelpViewProvider implements vscode.WebviewViewProvider {
+  private _view?: vscode.WebviewView;
+
+  constructor(private readonly _extensionUri: vscode.Uri) {}
+
+  public resolveWebviewView(
+    webviewView: vscode.WebviewView,
+    _context: vscode.WebviewViewResolveContext,
+    _token: vscode.CancellationToken
+  ): void {
+    this._view = webviewView;
+
+    webviewView.webview.options = {
+      enableScripts: true,
+    };
+
+    const onDiskPath = vscode.Uri.file(
+      path.join(__filename, '..', 'help_page.js')
+    );
```

Comment:
To make this work in both browser and node, you'll need to swap this for a platform neutral path utility, like here:

https://github.com/malloydata/malloy-vscode-extension/blob/main/src/extension/browser/commands/edit_connections.ts#L50-L54

You have `context`, so you can use `context.extensionUri` instead of the stowed value in `MALLOY_STATE`

Reply:
The _context is of `vscode.WebviewViewResolveContext` type. It has a `state` variable but no `extensionUri`.
I changed it to the private member variable `_extensionUri` of class `HelpViewProvider`.